chorus_lib/src/core.rs

```diff
@@ -123,17 +134,17 @@ pub trait ChoreoOp {
     ///
     /// `broadcast` broadcasts `data` from `sender` to all other locations. The `data` must be a `Located` struct at the `sender` location.
     /// The method returns the non-located value.
-    fn broadcast<L1: ChoreographyLocation, V: ChoreographicValue>(
+    fn broadcast<L1: ChoreographyLocation, V: Portable>(
         &self,
         sender: L1,
-        data: &Located<V, L1>,
+        data: Located<V, L1>,
```

Comment:
`broadcast` unwraps the value at the sender, so we need to pass the ownership of the data.